modules/importer/src/main/scala/ConfigReader.scala

```diff
@@ -1,308 +0,0 @@
-package gem
```

Comment:
These are part of the old `importer` module.  It's been replaced with a version that doesn't rely upon having access to OCS2 classes.